### DISCUSSION TITLE
Parameter to ignore workspaces on npm install

### DISCUSSION THREAD

**Discussion by tiagoporto** [npm/feedback#542] [CLI] [answered]
I have the following monorepo structure with workspaces set.

```
.
├─ package.json
│
├─ package-a
│        └─ package.json
│
└─ package-b
        └─ package.json
```

I would like to run `npm install` in the root only, and ignore the install in the **package-a** and the **package-b**.

A parameter like `--ignore-workspaces` could be helpful.

`npm install`: Install **root** dependencies and **package-a** and **package-b** dependencies.

`npm install --ignore-workspaces`: Install **root** dependencies only.

**Comment by MylesBorins** [comment, marked as answer]
@tiagoporto thanks for opening this discussion. If you would like to propose a CLI feature the next appropriate step would be to open an RFC issue over at https://github.com/npm/RFCS

**Comment by tobz1000** [comment]
This is possible with `npm install --workspaces=false`: https://docs.npmjs.com/cli/v10/commands/npm-install#workspaces
